integration/src/wallets/metamask.ts

```diff
@@ -83,23 +87,26 @@ export function selfTest(get: () => core.HDWallet): void {
 
   it("uses correct eth bip44 paths", () => {
     if (!wallet) return;
-    [0, 1, 3, 27].forEach((account) => {
+    // MM doesn't support multi-account
+    const accountNumbers = metamask.isMetaMask(wallet) ? [0] : [0, 1, 3, 27];
+    if (metamask.isMetaMask(wallet)) return;
```

Comment:
revert. no multi account wallet support, but can still get and describe paths as it stands.

Reply:
df02ebb2